pingdom-checks/veteran-verification.ping

```diff
@@ -1,4 +1,4 @@
-VETERAN_VERIFICATION_SLACK_ID=$SLACK_VASDVP_MONITORING
+VETERAN_VERIFICATION_SLACK_ID="${SLACK_VASDVP_MONITORING}, ${SLACK_API_ALARMS_VETERAN_VERIFICATION}"
```

Comment:
Not sure if the white space will affect the deployment or not, but it's best to remove it just in case.

Reply:
I followed this based on VA Letter Generator and it seems to have the same thing.